qiita_db/job.py

```diff
@@ -92,6 +93,10 @@ def exists(cls, datatype, command, options, analysis,
             Options for the command in the format {option: value}
         analysis : Analysis object
             The analysis the job will be attached to on creation
+        input_file_reference_id : int
```

Comment:
Because it's easier and I didn't see any strong evidence that doing string or object arguments is better, at least in this case ... BTW it also reduces SQL calls.